src/styles/index.scss

```diff
@@ -1,3 +1,134 @@
 body {
   margin: 0;
 }
+
+.product {
+  &-card {
+    padding-left: 16px;
+    display: flex;
+    box-sizing: border-box;
+    width: 200px;
+    flex-direction: column;
+    text-align: center;
+    font-family: Roboto, sans-serif;
+    border: 1px solid #f3f3f3;
+    border-radius: 5px;
+  }
+
+  &-box {
+    width: 166px;
+    margin-inline: auto;
+  }
+
+  &-image {
+    display: block;
+    margin: 32px auto 40px ;
+    width: 160px;
+    height: 134px;
+  }
+
+  &-price {
+    margin-top: 24px;
+    display: flex;
+    height: 18px;
+    justify-content: space-between;
+  }
+
+  &-calculation-price {
+    font-size: 16px;
+    line-height: 18px;
+    font-weight: 700;
+    color: #060b35;
+  }
+
+  &-button {
+    display: flex;
+    margin: 16px 0;
+    box-sizing: border-box;
+    background-color: #00acdc;
+    width: 100%;
+    height: 40px;
+    color: #fff;
+    font-size: 14px;
+    line-height: 16px;
+    font-weight: 700;
+    text-transform: uppercase;
+    justify-content: center;
+    align-items: center;
+    border: 1px solid #00acdc;
+    border-radius: 5px;
+    text-decoration: none;
+  }
+
+  &-button:hover {
+    background-color: #fff;
+    color: #00acdc;
+  }
+}
+
+.product-stars {
+  margin-top: 16px;
+  display: flex;
+  height: 16px;
+  justify-content: space-between;
+}
+
+.stars {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.stars__item {
+  height: 16px;
+  width: 16px;
+  margin: 0 4px 0 0;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-image: url(../images/star.svg);
+}
+
+.stars__element {
+  height: 16px;
+  width: 16px;
+  margin: 0 4px 0 0;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-image: url(../images/star-active.svg);
+}
```

Comment:
Consider using styles for stars block from the previous task

Reply:
I used styles for stars block from the previous task